native-gradle-plugin/src/samples/java-application-with-reflection/src/main/java/org/graalvm/demo/Message.java

```diff
@@ -0,0 +1,5 @@
+package org.graalvm.demo;
+
+public class Message {
+    public static final String MESSAGE = "Hello, native!";
+}
```

Comment:
I would honestly prefer not to. The less checkstyle rules we have, the better I feel. I would prefer rules encded in the IDE, checked in, so that we don't have to fail builds if the rules are not followed.

Reply:
Well we could - after we reorganize the repo to have a root project - add a default IDE configuration there. That way we would be able to both guard against improper style and be convenient (this is similar to what we do for the GraalVM itself with the style gates and `mx eclipseformat` and `mx ideinit` tools).